src/email/handleSubmitEmail.js

```diff
@@ -2,12 +2,14 @@ import { sendNotification } from './sendmail';
 import { logerror } from '../utils/logger';
 
 const CONFIRMATION_EMAIL = 'af563da5-43bd-4b9a-9610-e3990a7f4315'
-  
+
 
 
 export const handleSubmitEmail = async (req, res) => {
   let input = Object.assign({}, req.body) // make a new object
   if (input.templateId === CONFIRMATION_EMAIL){
+    console.log("handleSubmitEmail Line 11:"+JSON.stringify(input))
+    console.log("handleSubmitEmail Line 12:"+input.accessibility)
```

Comment:
Is this not throwing an error with es-lint? This looks like debugging logs.

Reply:
Good catch!